content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -45,6 +45,15 @@ on **each** Kubernetes cluster.
 
 ## Deploy the Istio control plane in each cluster {#deploy-istio}
 
+{{< tip >}}
```

Comment:
Wouldn't this be better in the prerequisites section, part of the `The IP address of the istio-ingressgateway service in each cluster must be accessible from every other cluster.` bullet? It should also be regular text. I've been trying to clean up some of the tips and warnings in several of our documents, since they are supposed to be used sparingly as mentioned in the last paragraph [here](https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts).

Reply:
Using an NLB is not a prerequisite.  It is more like a recommendation, but I'll add the text there as a new bullet.

Cheers
-steve